src/main/java/core/rating/RatingPredictionParameter.java

```diff
@@ -108,7 +108,7 @@ public double getParam (String section, String key, double defVal) {
     	if (allProps.containsKey(section)) {
     		Properties props = allProps.get(section);
     		String propString = props.getProperty(key, "" + defVal);
-    		if (propString != "") {
+    		if (!Objects.equals(propString, "")) {
```

Comment:
or `!propString.equals("")`

Reply:
changed to if (!propString.isEmpty()) {...

this class also might be removed, if we remove RatingPredictionManager. It could be useful if we implement the groovy api to override the new RatingPredictionModel. One candidate of that could be an implementation of the current rating prediction.